run.sh

```diff
@@ -229,6 +229,7 @@ select_smalltalk() {
                 GemStone64-3.5.0 GemStone64-3.4.3 GemStone64-3.3.9
                 GemStone64-3.3.2 GemStone64-3.3.0 GemStone64-3.2.12
                 GemStone64-3.1.0.6
+                GToolkit-release
```

Comment:
For consistency, should we call this `GToolkit-stable` or `GToolkit-trunk`? If not, I'd prefer `GToolkit-latest` over `GToolkit-release`.

Reply:
I called it "GToolkit-release" because that's what GT calls it. See https://dl.feenk.com/gt/. However, I don't have a strong opinion here and considered what you suggested as well. If you still think that after checking the link, I can make the change.